src/api/app/models/user.rb

```diff
@@ -448,6 +448,7 @@ def has_global_permission?(perm_string)
     end
   end
 
+  # FIXME: This should be a policy
```

Comment:
All the TODO here makes me wonder if we couldn't  call the Pundit Policy in the background without dropping those methods to avoid breaking API.

NITPICK: just wondering if it shouldn't be done in another PR